server/src/main/java/com/cloud/server/StatsCollector.java

```diff
@@ -477,16 +572,10 @@ protected void runInContext() {
                 SearchCriteria<HostVO> sc = _hostDao.createSearchCriteria();
```

Comment:
I have seen this before. Can you reuse it?

Reply:
Good catch! Done